packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7189,12 +7198,18 @@ export default class Meeting extends StatelessWebexPlugin {
    * @returns {number} time left in seconds
    */
   public getPermissionTokenTimeLeftInSec(): number | undefined {
-    if (!this.permissionTokenTtl) {
+    if (!this.permissionTokenPayload) {
       return undefined;
     }
 
-    // substract current time from the expiration time
-    // (permissionTokenTtl is a epoch timestamp, not a time to live duration)
-    return (this.permissionTokenTtl - Date.now()) / 1000;
+    const permissionTokenExpValue = Number(this.permissionTokenPayload.exp);
```

Comment:
nitpick: This should be a number already, but it doesn't hurt to ensure it anyway.